src/Entity/OgMembershipType.php

```diff
@@ -42,4 +59,35 @@ public function id() {
     return $this->type;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function save() {
+    $status = parent::save();
+
```

Comment:
This will result in #441, we need to check if config is syncing before creating this:

```suggestion

    if (\Drupal::isConfigSyncing()) {
      // Do not create data while config import in progress.
      return;
    }
```

Reply:
Thank you for pointing that out, committed.

Reply:
@Insasse not sure what happened but it seems like this commit is no longer part of this PR?
```suggestion
      if (\Drupal::isConfigSyncing()) {
        // Do not create config while config import is in progress.
        return;
      }
```

Reply:
This had been fixed.